src/main/java/com/rabbitmq/client/impl/AMQChannel.java

```diff
@@ -135,6 +137,20 @@ public AMQCommand exnWrappingRpc(Method m)
         }
     }
 
+    public CompletableFuture<Command> exnWrappingAsyncRpc(Method m)
```

Comment:
Since this is a public method, I'd like to clarify the name. How about `exceptionWrappingAsyncRpc`?